src/main/java/rs/teslaris/core/model/document/Document.java

```diff
@@ -69,7 +69,7 @@ public abstract class Document extends BaseEntity {
     private String scopusId;
 
     @ManyToOne(fetch = FetchType.LAZY)
-    @JoinColumn(name = "event_id")
+    @JoinColumn(name = "event_id", referencedColumnName = "id")
```

Comment:
Why this is the part of this PR? It is not connected to upgrade of spring boot, right?

Reply:
It is, I had some persistence bugs because new Hibernate was handling relationships a little bit different, I also followed some reccomendations so I added that.